src/contracts/core/AVSDirectory.sol

```diff
@@ -48,21 +56,200 @@ contract AVSDirectory is
         _transferOwnership(initialOwner);
     }
 
-    /*******************************************************************************
-                            EXTERNAL FUNCTIONS 
-    *******************************************************************************/
+    /**
+     *
+     *                         EXTERNAL FUNCTIONS
+     *
+     */
+
+    /**
+     * @notice Updates the standby parameters for an operator across multiple operator sets.
+     * Allows the AVS to add the operator to a given operator set if they are not already registered.
+     *
+     * @param operator The address of the operator for which the standby parameters are being updated.
+     * @param standbyParams The new standby parameters for the operator.
+     * @param operatorSignature If non-empty, the signature of the operator authorizing the modification.
+     *                  If empty, the `msg.sender` must be the operator.
+     */
+    function updateStandbyParams(
```

Comment:
This needs to be deeper in the file -- functions at the top are registration/deregistration. "update" methods should be below those for the easiest reading experience